### DISCUSSION TITLE
名字打错了

### DISCUSSION THREAD

**Discussion by XinRenShenJiangZiShu** [share121/inter-knot#4846] [其他]
怎么改名字啊，找不到啊

**Comment by wodedunpaifeichangying** [comment]
去github官网看看应该行？

**Reply by XinRenShenJiangZiShu** [reply]
找不到在那

**Comment by HLXS-HLXS** [comment]
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/changing-your-github-username#changing-your-username

**Comment by XinRenShenJiangZiShu** [comment]
感谢
